Code/GraphMol/Wrap/MolOps.cpp

```diff
@@ -1575,6 +1575,23 @@ struct molops_wrapper {
                  python::arg("flagPossibleStereoCenters") = false),
                 docString.c_str());
 
+    // ------------------------------------------------------------------------
+    docString =
+        "Uses a conformer to assign ChiralTypes to a molecule's atoms and\n\
```

Comment:
might specify 3D conformer as this won't work with 2D?

Reply:
certainly doesn't  hurt to be explicit.

Reply:
I will also add a check for that in the code itself.